test/hack/bin/check-opsrecipes.sh

```diff
@@ -6,9 +6,11 @@ RULES_FILES=(./test/hack/output/*/*/prometheus-rules/templates/alerting-rules/*)
 #RULES_FILES=(./test/hack/output/*/prometheus-rules/templates/alerting-rules/up*)
 
 DEBUG_MODE=false
+
 CHECK_EXTRADATA_ERRORS=true
 CHECK_NORECIPE_ERRORS=true
 CHECK_UNEXISTINGRECIPE_ERRORS=true
+OPSRECIPES_DIR=./giantswarm
```

Comment:
Ok so this is doing what I want it to but I do not like this. @hervenicol I think I need your help tomorrow